### PR TITLE
Update all dependencies

### PR DESCRIPTION
**Pull request by rianniello**
- Pins Rails to <= 5.2
- Unpins rails_admin
- Upgrades all other gems, including aaf-secure_headers, aaf-lipstick.

### CODE REVIEW

**Comment by rianniello** [comment]
Can we stall the merge please, just found Chrome misbehaves when viewing PDFs (may be CSP related). 

image

FF is fine.

**Comment by chrisjones120** [comment]
It's always a treat working with CSP.

**Comment by smangelsdorf** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=271452

Sounds like the answer might be to just disable CSP for the endpoint that serves the PDF.

**Comment by rianniello** [comment]
@smangelsdorf went through this earlier with @rcaught, can't be done without huge amounts of pain as the PDF rendering happens in its own middleware, i.e. https://github.com/ausaccessfed/validator-service/blob/076d514b76c4262a67001f9d4a99ca9fdf5a4bbc/config/application.rb#L27-L32 so we can't really get access to CSP headers.

Alternative is to just force the PDF as a download: https://github.com/ausaccessfed/validator-service/pull/178/commits/eacb6ccc214bef0545e5552a18888bf234d38662

File: config/application.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ class Application < Rails::Application
     config.shib_rack.error_handler = 'Authentication::ErrorHandler'
 
     app_config = YAML.safe_load(Rails.root.join(
-      'config', 'validator_service.yml'
```
Comment (rcaught) [inline config/application.rb]: There is no point in `.join` if you join them in the string.

File: config/environments/development.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
   config.consider_all_requests_local = true
 
-  if Rails.root.join('tmp', 'caching-dev.txt').exist?
+  if Rails.root.join('tmp/caching-dev.txt').exist?
```
Comment (rcaught) [inline config/environments/development.rb]: `join` again.

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -12,7 +12,7 @@
 
 ActiveRecord::Migration.maintain_test_schema!
 
-Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
+Dir[Rails.root.join('spec/support/**/*.rb')].sort.each { |f| require f }
```
Comment (rcaught) [inline spec/rails_helper.rb]: `join` again.

File: config/application.rb (hunk 1 above)
Comment (vladimir-mencl-eresearch) [reply config/application.rb]: I thought it's a join of Rails.root and the (single) path component given - so a join would still make sense?

Comment (rcaught) [reply config/application.rb]: ha!  So right @vladimir-mencl-eresearch.

**Pull request merged**